src/main/java/co/inventorsoft/academy/model/Match.java

```diff
@@ -0,0 +1,54 @@
+package co.inventorsoft.academy.model;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.experimental.FieldDefaults;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Getter
+@Setter
+@ToString(of = "id")
+@Table(name = "matches")
+@FieldDefaults(level = AccessLevel.PRIVATE)
+@RequiredArgsConstructor
```

Comment:
I see here no final fields.

Reply:
Why?

Reply:
In all guide I don't see final in Entity.

Reply:
And you don't use https://github.com/dgray16/inventorsoft-academy/blob/master/hibernate/src/main/java/com/inventorsoft/domain/model/Book.java

Reply:
`@RequiredArgsConstructor` creates constructor based on final class fields (which are not present in you class).
So this annotation is redundant.